ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala

```diff
@@ -359,7 +360,7 @@ object BigDiffy extends Command with Serializable {
     keyFn: TableRow => MultiKey,
     diffy: TableRowDiffy,
     ignoreNan: Boolean = false
-  ): BigDiffy[TableRow] = {
+  )(implicit coder: Coder[TableRow]): BigDiffy[TableRow] = {
```

Comment:
you shouldn't have to summon an implicit coder here -- since you're being explicit about using `TableRowJsonCoder.of()` below :)